app/controllers/api/v1/merchants/most_items_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Api::V1::Merchants::MostItemsController < ApplicationController
+  def index
+    #should be in model vvvvv
+    merchants = Merchant.joins(:transactions).
+    where(transactions: { result: 'success' }).
+    select('merchants.*, sum(invoice_items.quantity) as total_quantity').
+    group(:id).
+    order(total_quantity: :desc).
+    limit(params[:quantity])
```

Comment:
In the future this needs to transfer to the merchant model to live as a class method.